Makefile

```diff
@@ -3,7 +3,6 @@ MKFILE_DIR := $(dir $(abspath $(lastword $(MAKEFILE_LIST))))
 CPP_DIR := $(MKFILE_DIR)timemachine/cpp/
 INSTALL_PREFIX := $(MKFILE_DIR)timemachine/
 PYTEST_CI_ARGS := --cov=. --cov-report=term-missing
-BLACK_FLAGS := --extend-exclude 'ff/params/|parallel/grpc/' --line-length 120 .
```

Comment:
With this removed should remove the `make fmt` command

Reply:
good call, I missed this. Fixed in c9bb1e1.